definitions/immuno.wdl

```diff
@@ -199,6 +199,7 @@ workflow immuno {
     Array[Array[String]] gatk_haplotypecaller_intervals
     Int? ploidy
     String? optitype_name
+    Float germline_filter_gnomAD_maximum_population_allele_frequency
```

Comment:
Should immuno go ahead and set a default at the top level of 1.0 so that no variants get filtered?

Reply:
Testing now! I left germline_filter_gnomAD_maximum_population_allele_frequency set to 0.05  in [germline_detect_variants](https://github.com/wustl-oncology/analysis-wdls/blob/d952ad5c29f1f3aa400e15664e49c4ed0fd4c0ad/definitions/subworkflows/germline_detect_variants.wdl#L40). Would it be more correct style wise to set that value at the top level and have it be passed down?

Reply:
I think our general style is to define the defaults as low down as possible (at the tool level), and then pass parameters upward as needed so that they can be overridden (with the override being set as high up as possible, so we don't have to go digging through 8 layers to find it). In the top-level immuno wdl, I think it would be reasonable to set that value such that it always passes all germline variants, since various steps in that workflow depend on it.